src/api/api.ts

```diff
@@ -52,6 +52,9 @@ export class Api {
     const response = await lastValueFrom(
       getBackendSrv().fetch({
         method: 'GET',
+        headers: {
+          Accept: 'application/rss+xml',
```

Comment:
Data Source supports RSS and Atom feedback. For Atom the Content-Type is `application/atom+xml`. With this change we may break it?